app/models/custom_hint.rb

```diff
@@ -0,0 +1,87 @@
+# Loads our handcrafted hints. We collaborate on these in a google sheet, but
+# we draw them from a separate CSV file rather than automatically from Sheets
+# for two reasons:
+#  1) to avoid having a production app have reliance on a developer's personal
```

Comment:
I'm not sure if this is confusing because of the rebase, but it looks like this is still here in the final commit?